bouture.js

```diff
@@ -1,12 +1,43 @@
+const attributesName = ['accept' , 'accept-charset' , 'accesskey' , 'action' , 'align' , 'alt' , 'async' , 'autocomplete' , 'autofocus' , 'autoplay' , 'bgcolor' , 'border' , 'buffered' , 'challenge' , 'charset' , 'checked' , 'cite' , 'class' , 'code' , 'codebase' , '.class' , 'color' , 'cols' , ' textarea' , 'colspan' , 'content' , 'http-equiv' , 'name' , 'contenteditable' , 'contextmenu' , 'controls' , 'coords' , 'crossorigin' , 'data' , 'data-*' , 'datetime' , 'default' , 'defer' , 'dir' , 'dirname' , 'disabled' , 'download' , 'draggable' , 'dropzone' , 'enctype' , 'POST' , 'for' , 'form' , 'formaction' , 'action' , 'headers' , '<th>' , 'height' , 'hidden' , 'high' , 'href' , 'hreflang' , 'http-equiv' , 'icon' , 'id' , 'integrity' , 'ismap' , 'itemprop' , 'keytype' , 'kind' , 'label' , 'lang' , 'language' , 'list' , 'loop' , 'low' , 'manifest' , 'max' , 'maxlength' , 'minlength' , 'media' , 'method' , 'GET' , 'POST' , 'min' , 'multiple' , 'email' , 'file' , 'muted' , 'name' , 'novalidate' , 'open' , 'optimum' , 'pattern' , 'ping' , 'placeholder' , 'poster' , 'preload' , 'radiogroup' , 'readonly' , 'rel' , 'required' , 'reversed' , 'rows' , 'rowspan' , 'sandbox' , 'scope' , 'scoped' , 'seamless' , 'selected' , 'shape' , 'size' , 'type' , 'text' , 'password' , 'sizes' , 'slot' , 'span' , 'spellcheck' , 'src' , 'srcdoc' , 'srclang' , 'srcset' , 'start' , 'step' , 'style' , 'summary' , 'tabindex' , 'target' , 'title' , 'type' , 'usemap' , 'value' , 'width' , 'wrap']
 
-tags.forEach(tag => {
+tagsName.forEach(tag => {
   Object.defineProperty(Bouture, tag, {
     get: () => {
       const element = document.createElement(tag)
-      function branche (text) {
-        element.append(text)
+      function branche (...args) {
+        const append = {
+          string: (element, text) => element.append(text),
+          object: (element, attributes) => Object.keys(attributes)
+            .forEach(attribute => {
+              function setAttribute () {
+                return {
+                  boolean: {
+                    true: () => element.setAttribute(attribute, ''),
+                    false: () => element.removeAttribute(attribute)
+                  },
+                  string: {
+                    true: () => element.setAttribute(attribute, attributes[attribute]),
+                    false: () => element.setAttribute(attribute, '')
+                  },
+                  object: {
+                    true: () => attributes[attribute].forEach(each => {
+                      if (each) {
+                        element.classList.add(each)
+                      }
+                    })
+                  }
+                }
+              }
+              if (attributesName.indexOf(attribute) !== -1 || attribute.match(/^data-[a-z0-9]*$/)) {
+                setAttribute()[typeof attributes[attribute]][!!attributes[attribute]]()
```

Comment:
I find this line a bit hard to read
I think the code after line 13 would be more easily readable if it was a sequence of `if(typeof attributes[attribute] === ...) else ...`